kolibri/core/content/utils/channel_import.py

```diff
@@ -469,7 +469,12 @@ def check_and_delete_existing_channel(self):
                         new_channel_version=self.channel_version,
                     )
                 )
-                self.delete_old_channel_data(existing_channel.root.tree_id)
+
+                root_node = self.destination.session.query(
```

Comment:
is a regression test needed for this?

Reply:
Possibly, I've opened a follow up issue here: https://github.com/learningequality/kolibri/issues/5960